sqlmesh/core/engine_adapter/spark.py

```diff
@@ -104,8 +104,8 @@ def _sqlglot_to_spark_primitive_mapping(self) -> t.Dict[t.Any, t.Any]:
             exp.DataType.Type.DATE: spark_types.DateType,
             exp.DataType.Type.DATETIME: spark_types.TimestampNTZType,
             exp.DataType.Type.TIMESTAMPLTZ: spark_types.TimestampType,
-            exp.DataType.Type.TIMESTAMPTZ: spark_types.TimestampType,
             exp.DataType.Type.TIMESTAMP: spark_types.TimestampType,
+            exp.DataType.Type.TIMESTAMPTZ: spark_types.TimestampType,
```

Comment:
This change was triggered by https://github.com/tobymao/sqlglot/commit/07fa69dcb8970167ba0c55fff39175ab856ea9f3; without it, this test fails:

```
>       assert actual == expected
E       AssertionError: assert {'col_timesta...pe.TIMESTAMP)} == {'col_timesta...nested=False)}
E
E         Differing items:
E         {'col_timestamp': DataType(this=Type.TIMESTAMP)} != {'col_timestamp': DataType(this=Type.TIMESTAMPTZ, nested=False)}
E         Use -v to get more diff

tests/core/engine_adapter/test_spark.py:510: AssertionError
```